tests/samples/pp_custom_pformat.py

```diff
@@ -0,0 +1,17 @@
+
+def test():
+    x = 10
+    y = 20
+    assert pp(pp(x + y) + y) == 50
+    assert pp.deep(lambda: (x + y) + y) == 50
+    pp([14, 15, 16])
+
+    with snoop:
+        z = 30  # custom pformat is not used here
```

Comment:
Does this add anything to the test?

Reply:
Yes and no :) If we start using `pformat` for `snoop` this test will remind us of this change.